migrator/migrations/n_01_to_n_02_postgres_clusters/migration.go

```diff
@@ -19,10 +19,14 @@ import (
 	"gorm.io/gorm"
 )
 
+const (
+	startingSeqNum = pkgMigrations.BasePostgresDBVersionSeqNum() + 1, // 112
+)
+
```

Comment:
This appears to be a change to the autogenerated migration code, is there a functional reason why this change is being done? Or is it to align the migrations with the form we would like to maintain going forward